views/app.html

```diff
@@ -0,0 +1,173 @@
+      <div class="col-lg-12 desktop-install-app">
+        <div class="col-lg-3 desktop-install-app-btn-wrapper pull-left">
+          <button class="btn btn-block desktop-install-app-btn">{{ gettext("Get App")}}</button>
+        </div>
+        <div class="col-lg-12 send-install-link-container hidden pull-left">
+          <div class="col-lg-12 phone-number-label pull-left">
+            <div class="col-lg-4 pull-left">
+              <i class="fa icon-mobile-phone"></i> {{ gettext("Enter your phone number")}}
+            </div>
+          </div>
+          <div class="col-lg-4 pull-left">
+            <input class="phone-num-input" type="tel" name="" value="" placeholder="+1 (555) 555-5555">
+          </div>
+          <div class="col-lg-4">
+            <button class="btn btn-block send-sms-btn">{{ gettext("Send Text")}}</button>
+          </div>
+          <div class="col-lg-12 message-sent-container hidden pull-left">
+            <span>{{ gettext("message sent") }}</span>
+          </div>
+          <div class="col-lg-12 message-sent-error-container hidden pull-left">
+            <span>{{ gettext("message sent error") }}</span>
+          </div>
+        </div>
+      </div>
+    </div>
+    <div class="col-xs-6 col-xs-offset-3 app-download-button">
+      <button class="btn btn-block download-app-btn">{{ gettext("Download Webmaker")}}</button>
+    </div>
+    <div class="col-xs-8 col-xs-offset-2 col-lg-4 col-lg-pull-6 col-lg-offset-1 cellphone">
+      <img src="/img/cellphone.svg" alt="Cellphone">
+    </div>
+  </div>
+</div>
+<div class="content build-something">
+  <div class="row">
+    <div class="col-xs-8 col-xs-offset-2 col-lg-5 col-lg-offset-1 section-content push-down">
+      <div class="col-xs-12 section-header">{{ gettext("Build the Perfect App") }}</div>
+      <div class="col-xs-12 col-lg-8 section-text">{{ gettext("Exactly what you need") }}</div>
+    </div>
+    <div class="col-xs-10 col-xs-offset-1 col-lg-4 col-lg-offset-0 cellphone">
+      <img src="/img/cellphone.svg" alt="Cellphone">
+    </div>
+  </div>
+</div>
+<div class="content share-something">
+  <div class="row">
+    <div class="col-xs-10 col-xs-offset-1 col-lg-6 col-lg-offset-0 col-lg-push-6 section-content push-down">
+      <div class="col-xs-12 section-header">{{ gettext("Share It with the World") }}</div>
+      <div class="col-xs-12 col-lg-8 section-text">{{ gettext("Meant to be seen") }}</div>
+    </div>
+    <div class="col-xs-8 col-xs-offset-2 col-lg-4 col-lg-pull-6 col-lg-offset-1 cellphone">
+      <img src="/img/cellphone.svg" alt="Cellphone">
+    </div>
+  </div>
+</div>
+<div class="content download-app">
+  <div class="row">
+    <div class="col-xs-8 col-xs-offset-2 app-download-button">
+      <button class="btn btn-block download-app-btn">{{ gettext("Download Webmaker")}}</button>
+    </div>
+  </div>
+</div>
+
+<footer class="footer install-footer">
```

Comment:
> Can you put footer in footer.html inside views/components/ directory? so that can be reuse if we don't already have one :)

That's the angular footer, so it won't work. also, the footer for non-angular is shoved into the layout template, which I can't use for this